### PR TITLE
Refactoring unicode to codePoint

### PR DESCRIPTION
**Pull request by ollimeier**
Fixes #1105 in fontra

Related to:
- https://github.com/googlefonts/fontra/issues/1105
- https://github.com/googlefonts/fontra/pull/1139

### CODE REVIEW

File: src/fontra_rcjk/base.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 class GLIFGlyph:
     def __init__(self):
         self.name = None  # Must be set to a string before we can write GLIF data
-        self.unicodes = []
+        self.codePoints = []
```
Comment (justvanrossum) [inline src/fontra_rcjk/base.py:31]: This one should also not change, as this class also needs to be compatible with fontTools.ufoLib.

File: src/fontra_rcjk/backend_mysql.py [hunk 2]
```diff
@@ -510,7 +510,7 @@ async def _pollOnceForChanges(self) -> tuple[Any, Any]:
 
 
 def _codePointsFromGlyphInfo(glyphInfo: dict) -> list[int]:
-    return glyphInfo.get("codePoints", [])
+    return glyphInfo.get("unicodes", [])
```
Comment (ollimeier) [inline src/fontra_rcjk/backend_mysql.py:513]: @justvanrossum eg. I am not sure what do do with this line of code. Keep 'unicodes' or changing to 'codePoints'?

Comment (justvanrossum) [reply src/fontra_rcjk/backend_mysql.py:513]: This one needs to stay, since it "glyphInfo" is received from the server, and is not in a format we control.

File: src/fontra_rcjk/base.py (hunk 1 above)
Comment (ollimeier) [reply src/fontra_rcjk/base.py]: Thanks. done.

File: src/fontra_rcjk/backend_mysql.py (hunk 2 above)
Comment (ollimeier) [reply src/fontra_rcjk/backend_mysql.py:513]: Thanks. Mark as done.

File: src/fontra_rcjk/backend_mysql.py [hunk 3]
```diff
@@ -351,14 +351,14 @@ async def _newGlyph(self, glyphName: str, unicodes: list[int]) -> None:
         logger.info(f"Creating new glyph '{glyphName}'")
         dummyGlyph = GLIFGlyph()
         dummyGlyph.name = glyphName
-        dummyGlyph.unicodes = unicodes
+        dummyGlyph.codePoints = codePoints
```
Comment (justvanrossum) [inline src/fontra_rcjk/backend_mysql.py:354]:
```suggestion
        dummyGlyph.unicodes = codePoints
```

Comment (justvanrossum) [reply src/fontra_rcjk/backend_mysql.py:354]: This was caught by mypy...

File: README.md [hunk 4]
```diff
@@ -1,2 +1,23 @@
...
+2. Go to you fontra-rcjk folder/repo
+3. Install the required fontra branch via: `pip install -e ../fontra/`
+4. Run `pytest`
+
+Please keep in mind the *_fs backend has some unit tests. The mysql one does not, so we need to be super careful there.
+
+
+# How to run a rcjk project
+
+1. Start fontra like this: `fontra rcjk robocjk.black-foundry.com`
```
Comment (justvanrossum) [inline README.md:20]: This is not a public site, so we should not mention that.

Also: these changes are unrelated to the renaming of unicode(s) to codePoint(s), so should go to a different PR. Please remove this change from this PR.

**Comment by justvanrossum** [comment]
I merged https://github.com/googlefonts/fontra/pull/1139, and fixed one tiny bug, and now the tests pass.

File: README.md (hunk 4 above)
Comment (ollimeier) [reply README.md]: OK. Done.
